packages/private-build-infra/src/addon-build-config-for-data-package.js

```diff
@@ -54,69 +42,15 @@ function addonBuildConfigForDataPackage(pkg) {
       });
     },
 
-    _prodLikeWarning() {
-      let emberEnv = process.env.EMBER_ENV;
-      if (emberEnv !== 'production' && /production/.test(emberEnv)) {
-        this._warn(
-          `Production-like values for EMBER_ENV are deprecated (your EMBER_ENV is "${emberEnv}") and support will be removed in Ember Data 4.0.0. If using ember-cli-deploy, please configure your build using 'production'. Otherwise please set your EMBER_ENV to 'production' for production builds.`
-        );
-      }
-    },
-
     isDevelopingAddon() {
       if (typeof this.parent.name === 'string' && this.parent.name === 'ember-data') {
         return this.parent.isDevelopingAddon();
       }
       return this._super(...arguments);
     },
 
-    _warn(message) {
-      let chalk = require('chalk');
-      let warning = chalk.yellow('WARNING: ' + message);
-
-      if (this.ui && this.ui.writeWarnLine) {
-        this.ui.writeWarnLine(message);
-      } else if (this.ui) {
-        this.ui.writeLine(warning);
-      } else {
-        // eslint-disable-next-line no-console
-        console.log(warning);
-      }
-    },
-
-    _suppressUneededRollupWarnings(message, next) {
-      if (message.code === 'CIRCULAR_DEPENDENCY') {
-        return;
-      } else if (message.code === 'NON_EXISTENT_EXPORT') {
-        // ignore type imports
-        if (message.message.indexOf(`@ember-data/types`) !== -1) {
-          return;
-        }
-      } else if (message.code === 'UNRESOLVED_IMPORT') {
-        if (!this.isDevelopingAddon()) {
-          // don't print these for consumers
-          return;
-        } else {
-          const chalk = require('chalk');
-          // make warning actionable
-          // eslint-disable-next-line no-console
-          console.log(
-            chalk.yellow(
-              `\n\n⚠️  Add ${chalk.white(
-                message.source
-              )} to the array returned by externalDependenciesForPrivateModule in index.js of ${chalk.white(
-                this.name
-              )}\n\n`
-            )
-          );
-          throw message.message;
-        }
-      }
-      next(message);
-    },
-
     shouldIncludeChildAddon(addon) {
-      if (addon.name.startsWith('@ember-data')) {
+      if (addon.name.startsWith('@ember-data/')) {
```

Comment:
note this is not the broccoli node issue (see the `@`) but since it could manifest in a similar incorrect check should someone choose a github/npm org starting with @ember-data* we add the `/`